src/resources/cpp/language-basics.md

```diff
@@ -144,26 +136,43 @@ Assignment is a process that takes place after a variable already exists. It is
 an existing variable.
 
 ```cpp
-int foo = 0; // Initialization
-foo = 2;     // Assignment
+int foo = 0; // initialization (begins with the type of the variable)
+foo = 2;     // assignment (no type specified)
 ```
 
-Like initialization, assignment has several different forms that will be discussed in more dtail in a later chapter.
+Like initialization, assignment has several different forms that will be discussed in more detail in a later chapter.
 
 ## Fundamental Data Types and Literals
 
-Data types are a way in code to represent a unit of data. Programmers will build data types using other data types. C++
-provides developers with a set of data types built into the language, called fundamental types. Fundamental types cannot
-be split further without losing meaning. You can think of these types as the "atoms" of your programs.
+All values in C++ have a _type_, and variables can only hold values of a specified type.
+
+The most simple types are _fundamental types_. You have already seen variables of (fundamental) type `int` in the
+examples above, and values such as `0` (also of type `int`). We will go over some commonly used fundamental types in the
+following section.
 
-### Booleans
+### `void`
 
-The most fundamental type that can hold a value is a boolean. A boolean is a representation of `true` or `false`.
+`void` is a fundamental type that has no value. You can use it as the result of functions that only "do stuff", but have
```

Comment:
I'm not convinced that `void` is a necessarily topic to introduce yet. This may be better suited to an actual introduction to functions.

Reply:
In hindsight, agreed. I think I'll put this into another draft PR or something.